cmd/dagger/module.go

```diff
@@ -346,6 +346,42 @@ var moduleInstallCmd = &cobra.Command{
 	},
 }
 
+var moduleUnInstallCmd = &cobra.Command{
+	Use:     "uninstall [options] <module>",
+	Short:   "Uninstall a dependency",
+	Long:    "Uninstall module as a dependency from the current module. The target module must be local.",
+	Example: "dagger uninstall github.com/shykes/daggerverse/hello",
```

Comment:
I think you'd most likely want to uninstall by name rather than full ref, so the example should reflect that.

Reply:
done